### PR TITLE
roles/pulpito: support github pull branches to checkout

### PR DESCRIPTION
**Pull request by kshtsk**
Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@gmail.com>

### CODE REVIEW

File: roles/teuthology/vars/zypper_opensuse_leap_15.3.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (djgalloway) [inline roles/teuthology/vars/zypper_opensuse_leap_15.3.yml:1]: Why is this file in this PR?

Comment (kshtsk) [reply roles/teuthology/vars/zypper_opensuse_leap_15.3.yml:1]: I was testing it on the latest distro, so this is a complement.

Comment (djgalloway) [reply roles/teuthology/vars/zypper_opensuse_leap_15.3.yml:1]: Ah, right, you do a full pulpito/paddles/teuthology dev setup to test, right?

Comment (kshtsk) [reply roles/teuthology/vars/zypper_opensuse_leap_15.3.yml:1]: @djgalloway exactly

**Review by djgalloway — APPROVED** [review]

**Pull request merged**